psi4/src/psi4/libfock/jk.h

```diff
@@ -303,6 +314,21 @@ class PSI_API JK {
     /// wK matrices: wK_mn = (ml|w|ns) C_li^left C_si^right
     std::vector<SharedMatrix> wK_ao_;
 
+    /// D, J, K, wK Matrices from previous iteration, used in Incremental Fock Builds
+    std::vector<SharedMatrix> D_ao_prev_;
```

Comment:
Can you elaborate on why this needs to be a vector? For alpha and beta?

Reply:
Ah, yes, D_ao_ is also a vector of matrix pointers, so D_ao_prev_ would just be an analog to it (indeed for alpha and beta in UHF and ROHF).